### PR TITLE
Tidy up the public module level statics

### LINKED ISSUE #6024: No error and no visualization when using `Standard.Base.Data.Json.parse`
### Discord username

_No response_

### What type of issue is this?

Permanent – Ocurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

I'm trying to create such a graph. There is no error nor visualization:

image

### Expected behaviour

Either error or visualization. dead sd and asks fast and asd

### How we can reproduce it?

Create the graph from screenshot.

### Screenshots or screencasts

_No response_

### Enso Version

newest 

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

standalone

### Operating System

MacOS

### Operating System Version

_No response_

### Hardware you are using

_No response_

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

Tidies up a lot of PUBLIC module statics - marked some as PRIVATE, made some methods of types.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by farmaazon — APPROVED** [review]
Approved the JS file.

File: distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso [hunk 1]
```diff
@@ -1549,8 +1549,8 @@ type Table
    - connection: The connection to a database.
    - table_name: The name of the table to get.
    - columns: The names of the columns to get.
+     Pairs of name and type.
```
Comment (radeusgd) [inline distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso]:
```suggestion
   - columns: List of columns to fetch. Each column is represented by a pair of column name and its expected SQL Type.
```

File: distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso [hunk 2]
```diff
@@ -110,7 +110,8 @@ default_aggregate_column_name aggregate_column include_column=True =
             c = aggregate_column.column
             prefix + " " + (if include_column then c.name else "")
 
-## Utility function to check if all same column
+## PRIVATE
+   Utility function to check if all same column
```
Comment (radeusgd) [inline distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso]:
```suggestion
   Utility function to check if all aggregates are operating on the same source column.
```

Not related to this PR but this doc string was keeping confusing me.

File: test/Tests/src/Semantic/Case_Spec.enso [hunk 3]
```diff
@@ -110,10 +110,10 @@ spec = Test.group "Pattern Matches" <|
     Test.specify "should be able to match on the Polyglot type" <|
         random_gen = Random.new
         case random_gen of
-            Polyglot.Polyglot -> Nothing
+            Polyglot -> Nothing
             _ -> Test.fail "Expected a polyglot object to match."
-        case Polyglot.Polyglot of
-            Polyglot.Polyglot -> Nothing
+        case Polyglot of
+            Polyglot -> Nothing
```
Comment (radeusgd) [inline test/Tests/src/Semantic/Case_Spec.enso:116]: That indeed seems to look better.

**Review by radeusgd — APPROVED** [review]
Looks all good.

I'm especially glad of moving some of the internals to an `Internal` module.

**Review by hubertp — APPROVED** [review]

**Review by GregoryTravis — APPROVED** [review]

**Pull request merged**